seaborn/_core/subplots.py

```diff
@@ -21,12 +21,10 @@ class Subplots:
     ----------
     subplot_spec : dict
         Keyword args for :meth:`matplotlib.figure.Figure.subplots`.
-    facet_spec : dict
+    facet_spec : FacetSpec
```

Comment:
I don't think this is the right way to document the inputs here; `FacetSpec` is something `mypy` would understand but it's not something you would instantiate and pass. (This class is internal anyway and so probably the signature annotations suffice, and these types could be removed).

Reply:
Thanks for the explanation! I got it